client/SymbolClient.py

```diff
@@ -54,8 +54,36 @@ def __init__(self, address):
 		self.voting_public_keys = []
 
 
-class SymbolPeerClient:
+class SymbolLightRESTClient:
+	"""
+	Class for Symbol light rest client.
+	"""
+
+	def __init__(self, host, **kwargs):
+		self.node_host = host
+		self.session = create_http_session(**kwargs)
+
+	def is_ssl(self):
+		try:
+			url = f'https://{self.node_host}:3001/node/info'
+			self.session.get(url, timeout=5)
+			return True
+		except (RequestException, TimeoutError):
+			return False
+
+	def get_rest_version(self):
+		try:
+			json_http_headers = {'Content-type': 'application/json'}
+			json_response = self.session.get(f'http://{self.node_host}:3000/node/server', headers=json_http_headers).json()
+
+			return json_response['serverInfo']['restVersion']
+		except (RequestException, TimeoutError):
+			return None
+
+
+class SymbolPeerClient(SymbolLightRESTClient):
```

Comment:
Should ``SymbolPeerClient`` be the base class instead?

Reply:
`SymbolPeerClient ` - it is used to query data from the peer node (socket)
`SymbolClient` - it is used to query data from the API endpoint.

As I needed to collect `isSSL` and `rest_version` from the API node.
But at the same time, I need it for the light node (Peer) as well.

So that I created a `SymbolLightRESTClient`, that method can use between `SymbolPeerClient ` and `SymbolClient`

Maybe I should rename to something else

Reply:
Renaming the class will help, but I don't know if this is the right fit since you have added the rest method to the peer client.

Would it be better to check if client node [here](https://github.com/symbol/miscellaneous/blob/b07fe0c4d458987a462eff71fed5125ce411acbe/client/SymbolClient.py#L156) and return the ``SymbolClient`` or the ``SymbolLightRESTClient``?